src/peft/peft_model.py

```diff
@@ -2510,6 +2513,20 @@ def check_irrgular(vals: list[bool]) -> bool | Literal["irregular"]:
 
         requires_grad = {key: check_irrgular(vals) for key, vals in mapping_requires_grad_list.items()}
 
+        devices_dd = collections.defaultdict(list)
+        for adapter_module_name in module.adapter_layer_names + module.other_param_names:
+            adapter_module = getattr(module, adapter_module_name)
+            if isinstance(adapter_module, torch.nn.ModuleDict):
+                for key, submodule in adapter_module.items():
+                    devices_dd[key].extend([param.device.type for param in submodule.parameters()])
+            elif (
+                isinstance(adapter_module, torch.nn.ParameterDict)
+                or (adapter_module.__class__.__name__ == "BufferDict")  # VeRA
+            ):
```

Comment:
Should this be covered for DoRA too?

Reply:
Yes, this works for DoRA too. As DoRA is just a `LoraModel`, the shown status will be almost the same for LoRA and DoRA, except that DoRA has a few more parameters.